### ISSUE TITLE
Still false positive for font-families (value-keyword-case)

### ISSUE THREAD

**Issue by strarsis** [stylelint/stylelint#5154]
> Clearly describe the bug

When declaring a SCSS variable with common font-family names, stylelint rule `value-keyword-case` complains about it:
```scss
$font-primary: Tahoma, Verdana, Segoe, sans-serif;
````

> Which rule, if any, is the bug related to?
`value-keyword-case`

> What code is needed to reproduce the bug?
```scss
$font-primary: Tahoma, Verdana, Segoe, sans-serif;

body {
  font-family: $font-primary;
}
````

> What stylelint configuration is needed to reproduce the bug?

e.g.

```js
module.exports = {
  'extends': 'stylelint-config-standard',
  'rules': {
    'no-empty-source': null,
    'string-quotes': 'double',
    'at-rule-no-unknown': [
      true,
      {
        'ignoreAtRules': [
          'extend',
          'at-root',
          'debug',
          'warn',
          'error',
          'if',
          'else',
          'for',
          'each',
          'while',
          'mixin',
          'include',
          'content',
          'return',
          'function',
          'tailwind',
          'apply',
          'responsive',
          'variants',
          'screen',
        ],
      },
    ],
  },
};

```

> Which version of stylelint are you using?
`13.10.0`

> How are you running stylelint: CLI, PostCSS plugin, Node.js API?
`webpack (5; Laravel Mix)`

> Does the bug relate to non-standard syntax (e.g. SCSS, Less etc.)?
Yes, it's related to SCSS nested properties.

> What did you expect to happen?
No warnings to be flagged.

> What actually happened (e.g. what warnings or errors did you get)?
Expects the value to be in lowercase.

This issue has been reported and was apparently fixed multiple times.

**Comment by jeddy3** [comment]
@strarsis Thanks for the report and for using the template.

The rule looks to be behaving as designed. You'll need to [configure it for your needs, though](https://github.com/stylelint/stylelint/issues/4622#issuecomment-594641799).

**Comment by strarsis** [comment]
@jeddy3:
Adding the required option to `.stylelintrc` doesn't seem to fix the issue:
```js
{
  "extends": "stylelint-config-standard",
  "rules": {
    "declaration-colon-newline-after": null,
    "value-list-comma-newline-after": null,
    "no-empty-source": null,
    "no-descending-specificity": null,
    "at-rule-empty-line-before": null,
    "at-rule-no-unknown": [
      true,
      {
        "ignoreAtRules": [
          "extend",
          "at-root",
          "debug",
          "warn",
          "error",
          "if",
          "else",
          "for",
          "each",
          "while",
          "mixin",
          "include",
          "content",
          "return",
          "tailwind",
          "apply",
          "responsive",
          "variants",
          "screen",
          "function"
        ]
      }
    ],
    "value-keyword-case": ["lower", { "ignoreProperties": ["/^$/"] }]
  }
}
````
Setting `value-keyword-case` to `null`, disabling it completely works, but I only want to let it skip the lowercase check for variables.

**Comment by jeddy3** [comment]
My bad. You need to escape the dollar character:

```
"value-keyword-case": ["lower", { "ignoreProperties": ["/^\\$/"] }]
```

I've updated the comment in the other issue to reflect this.
